pkgs/data/fonts/behdad-fonts/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "behdad-fonts";
+  version = "0.0.3";
+
+  src = fetchFromGitHub {
+    owner = "font-store";
+    repo = "BehdadFont";
+    rev = "v${version}";
+    sha256 = "0rlmyv82qmyy90zvkjnlva44ia7dyhiyk7axbq526v7zip3g79w0";
+  };
+
+  installPhase = ''
+    mkdir -p $out/share/fonts/behdad-fonts
+    cp -v $( find . -name '*.ttf') $out/share/fonts/behdad-fonts
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/font-store/BehdadFont;
+    description = "A Persian/Arabic Open Source Font - بهداد: فونت فارسی با مجوز آزاد";
```

Comment:
I think we should stick with ASCII here.

Reply:
Why?
Is there any technical issue with it?

Reply:
Some non-ASCII are already in descriptions (and maintainer names), for example
https://github.com/NixOS/nixpkgs/blob/master/pkgs/data/fonts/caladea/default.nix#L26

Right-to-left script might help to spot a bug, in Hydra web UI or something like this